src/store/reducers/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { constants } from "../actions/constants";
+
+const initialState = {
+  error: "",
+  films: [],
+  loading: false,
+  sortParam: "release_date",
+  genre: "all",
+  movieId: "",
+  movie: {},
+};
+
+export const movieReducer = (state = initialState, action) => {
```

Comment:
```suggestion
export const movieReducer = (state = initialState, { type, payload }) => {
```
It could be shorter, but ok 😏

Reply:
I read that it's a good practise to create actions. I understand that it sounds funny for you that i want to follow good practises in specific places. But i not always know about them :)